openquake/hazardlib/geo/surface/kite_fault.py

```diff
@@ -156,6 +158,24 @@ def get_tor(self):
         iro = iro[iro <= 1]
         return self.mesh.lons[iro, ico], self.mesh.lats[iro, ico]
 
+    @property
+    def is_vertical(self):
+        """ True if all the profiles, and hence the surface, are vertical """
```

Comment:
Personally I use a property only if the operation is ultra-fast, otherwise I prefer to use a method. In this case I would remove the property since the operations that are performed underlying are a lot more complex than a simple attribute access.